docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
     environment:
       MONITOR_PORT_LOG: ${MONITOR_PORT_LOG}
       CONTROLLER: "${CONTROLLER_HOST}:${CONTROLLER_PORT_SSH}"
+      DB_CONNECTION: "mongodb://${DB_ROOT_USER:-root}:${DB_ROOT_PASSWORD:-root_password}@ocrd-database:27017"
```

Comment:
I don't want to be picky, but we already use `DB_*` in top-level .env for the MySQL database in Kitodo, and it clashes on `DB_ROOT_PASSWORD` (but not `DB_ROOT_USER`). Do we simply tie these two databases to the same credentials here, or rather create a separate set of variables?